ui/lib/core/addon/templates/components/form-field.hbs

```diff
@@ -71,7 +71,10 @@
       @onChange={{action (action "setAndBroadcast" valuePath)}} @inputValue={{get model valuePath}}
       @helpText={{attr.options.helpText}} @subText={{attr.options.subText}} @label={{labelString}}
       @subLabel={{attr.options.subLabel}}
-      @fallbackComponent={{attr.options.fallbackComponent}} />
+      @fallbackComponent={{attr.options.fallbackComponent}}
+      @initialSelected={{initialSelected}}
```

Comment:
I wonder if we should pass the initialSelected and selectLimit on the attr so it follows convention for how this component is used. Thoughts?

Reply:
The problem with that is initialSelected isn't known on the model (which is where attr.options comes from).  SelectLimit can be though.